encode-faq-titles.sh

```diff
@@ -1,15 +1,18 @@
 #!/bin/bash
 set -e
 
-if [ -z "${AUTH_PASSWORD}" ]; then
+if [ -z "${AUTH_PASSWORD}" ] && [[ "$@" != *"-local"* ]]; then
     echo "no AUTH_PASSWORD provided!"
     exit 1
 fi
 
-NLP_API_URL=https://stampy-nlp-t6p37v2uia-uw.a.run.app/api/encode-faq-titles
-# NLP_API_URL=https://test---stampy-nlp-t6p37v2uia-uw.a.run.app/api/encode-faq-titles
-# NLP_API_URL=http://localhost:8080/api/encode-faq-titles
-
-curl -X POST -f -u stampy:$AUTH_PASSWORD $NLP_API_URL -H 'Content-Type: application/json' > stampy-duplicates.json
+if [[ "$@" == *"-local"* ]]; then
+    source venv/bin/activate
+    pip install -e '.[local_model]'
+    RETRIEVER_MODEL_URL='./model_server/models/multi-qa-mpnet-base-cos-v1/' python src/stampy_nlp/faq_titles.py $@
```

Comment:
this model path works for me, should I change it to something else?

Reply:
RETRIEVER_MODEL_URL=multi-qa-mpnet-base-cos-v1

This would be a better general default. Your path only work if the model server has already been set up, but that may not necessarily be the case.